docs/checkouts/features.mdx

```diff
@@ -0,0 +1,93 @@
+  select the payment method (or tokenize), and complete the checkout.
+
+### Insurance Quote Payments
+
+- To include an insurance quote in Checkout, contact JustiFi to enable insurance payments for
+  your platform. Ensure the required insurance type is available through our provider.
+- After creating a Checkout, use the [Web Component with insurance](https://storybook.justifi.ai/?path=/story/payment-facilitation-unified-fintech-checkout%E2%84%A2--checkout-insurance)
+  to collect the insurance payment.
+
+### BNPL Payments
+
+- To enable BNPL payments, contact JustiFi to activate BNPL for your platform and Sub-Account.
+- After creating a Checkout, use the [Web Component](https://storybook.justifi.ai/?path=/story/payment-facilitation-unified-fintech-checkout%E2%84%A2--example)
+  with the BNPL option.
+
+_note: Disable the BNPL option by passing the correspondent prop to the web component:
+[disable-bnpl](https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs)_
+
+### Terminal (Card Reader) Payments
+
+- For terminal payments, contact JustiFi to enable the card-present feature.
+  JustiFi will then provision and configure terminals for your sub-accounts.
+- Find detailed steps in the [Terminals API](https://docs.justifi.tech/api-spec#tag/Terminals)
+
+### Application Fees
+
+- Application Fees are configured at the Platform level and can be customized for specific Sub-Accounts.
+  Checkouts allow flexibility by using the `application_fee_amont` parameter when [creating the payment](https://docs.justifi.tech/api-spec#tag/Payments/operation/CreatePayment)
+
+- **Create a Checkout:** Use the [Create Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts)
+   with a custom application fee amount for card and bank payments, as in the following example:
+
+```json
+{
+   "amount": 1000,
+   "description": "test checkout application fees",
+   "application_fees": {
+      "card": {
+         "amount": "200"
+      },
+      "bank_account": {
+         "amount": "100"
+      }
+   }
+}
+```
+
+- The application fee applies based on the selected payment method during checkout.
+_note: available for Card, ACH and Terminal (Card Present) only!_
+
+### Payment Method Group
+
+- **Create a Payment Method Group:** Use the [Payment Method Group API](https://docs.justifi.tech/api-spec#tag/Payment-Method-Groups)
+  to create a group for easy payment method access.
+- **Create a Checkout:** Use the [Create Checkout API](https://docs.justifi.tech/api-spec#tag/Checkouts/operation/CreateCheckout)
+  and pass the `payment_method_group_id`.
+- **Checkout Web Component:** Use the [Checkout Web Component](https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs)
+  with the created checkout. This allows the component to display a list of tokenized payment methods
```

Comment:
I'd update '... with the created checkout. This allows the component to display a list...' to '... and pass in the checkout id. This allows the web component to display the list ...'

Reply:
Oh yes, I missed this:
I'd change this bullet point to:
"**Checkout Web Component:** Use the [Checkout Web Component](https://storybook.justifi.ai/?path=/docs/payment-facilitation-unified-fintech-checkout%E2%84%A2--docs)
 and pass the checkout id. This allows the web component to display a list of tokenized payment methods"